### PR TITLE
fix so dropping on card has latest cards objects

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Fixes issue so that dropping on cards has latest card data and doesn't lose recent changed.

#### Ticket Link

  Fixes https://github.com/mattermost/focalboard/issues/4719

### CODE REVIEW

**Comment by sbishel** [comment]
@mgdelacroix - Any idea why this one works, https://github.com/mattermost/focalboard/pull/4726

But this one fails?

**Comment by mgdelacroix** [comment]
@sbishel there is a branch with the same name in `mattermost-server` for this one, so the CI instead of downloading the pre-mono-repo version, downloads the branch one which doesn't work. Either we wait until the server branch is merged and deleted (and some time more for the github cache to clear) or one of the branches needs to be deleted and reuploaded with a different name

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1: 

I've run both `npm run check` and `npm run test` and both seem to be working, so with that and the fact that the change is very small I'd say we can merge and fix the CI on `main` if something fails

**Comment by mgdelacroix** [comment]
CI on webapp `main` run successfully

**Pull request merged**
